assets/helpers/tracking/guTracking.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+// ----- Campaigns ----- //
+
+const baselineTestPrefix = 'gdnwb_copts_memco_sandc_support_baseline_support';
+
+const campaigns : {
+  [string]: string[],
+} = {
+  baseline_test: [
+    `${baselineTestPrefix}_banner`,
+    `${baselineTestPrefix}_epic`,
+    `${baselineTestPrefix}_liveblog`,
+    `${baselineTestPrefix}_header_become_supporter`,
+    `${baselineTestPrefix}_header_subscribe`,
+    `${baselineTestPrefix}_side_menu_become_supporter`,
+    `${baselineTestPrefix}_side_menu_subscribe`,
+  ],
+};
+
+export type Campaign = $Keys<typeof campaigns>;
+
+
+// ----- Functions ----- //
+
+// Checks if a user is in a known campaign based upon their intCmp.
+function inCampaign(campaign: Campaign, intCmp: string): boolean {
+
+  return campaigns[campaign] && campaigns[campaign].includes(intCmp);
+
+}
+
+// Retrieves the user's campaign, if known, from the intCmp.
+function getCampaign(intCmp: string): ?Campaign {
+
+  return Object.keys(campaigns).find(campaign =>
+    campaigns[campaign].includes(intCmp),
+  ) || null;
+
+}
+
+
+// ----- Reducers ----- //
+
+// Since nothing changes the INTCMP, this reducer does not handle any actions.
+const intCmpReducer = (state: ?string = null): ?string => state;
+
+const refpvidReducer = (state: ?string = null): ?string => state;
```

Comment:
Is it possible to combine these two reducers in one reducer called `guTrackingReducer` which state contains two fields? In that way, we would import only one reducer in other files and in that way have less code around.